grc/scripts/freedesktop/org.gnuradio.grc.metainfo.xml

```diff
@@ -50,11 +50,11 @@
 <url type="homepage">https://www.gnuradio.org</url>
 <url type="help">https://wiki.gnuradio.org/</url>
 <url type="bugtracker">https://github.com/gnuradio/gnuradio/issues</url>
-<url type="contact">mailto:discuss-gnuradio@gnu.org</url>
```

Comment:
:/ 

I kind of don't... like this change (don't heed me, I'm just a grumpy old guy), because this is something the metainfo schema checker incorrectly marks as error; It's perfectly [legal](https://freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-url) to use a `mailto:` URL, and it'd be honest, because, hey, that's how we as community actually communicate, for an important part. However, it's discouraged, by freedesktop, as Desktop environments seem to have a hard time making it easy for users to send webmail, and now that is a problem of software maintainers ;) I think they might be right though, if not in the stated reasoning, nobody looks at the software description page and then spontaneously wants to send an email.